app/src/lib/rules/rules_parser.js

```diff
@@ -387,7 +387,12 @@ export function create_token_transform(transform_json) {
 	 * @returns {Tag}
 	 */
 	function add_value_to_tag(old_tag, new_values) {
-		return { ...old_tag, ...new_values }
+		const result = { ...old_tag, ...new_values }
+		// completely remove tags that are being set as empty
```

Comment:
just wanted to double-check that this couldn't have been done earlier.  `add...`  function with remove functionality seems suspect to em at this level...

Reply:
I have considered adding a `remove_tag` option in the transform rules, but felt I wanted to wait until there was obvious need. I suppose now is that time!

Reply:
you can always save it until some of your next PR's if you want to get this one in.

Reply:
I like the idea of doing it now. Shouldn't take long